README.md

```diff
@@ -1,3 +1,60 @@
 # Silksong.ModMenu
 
-A Hollow Knight: Silksong mod.
+The standard menu-modification mod for Silksong. It provides a small but powerful set of features for mods to use.
+
+This mod also adds a 'Mod Options' menu to the main Options menu by default.
+
+The rest of this page is targeted towards mod developers. If you're not developing mods, you can stop reading now.
+
+## Mod Options
+
+This mod adds a 'Mods' sub-menu to the bottom of the Options menu as its main feature. All other plugins can add content to this menu as they wish.
+
+By default, any installed plugin's [`ConfigFile`](https://docs.bepinex.dev/master/api/BepInEx.Configuration.ConfigFile.html) data will be used to generate an ingame menu that can modify those configuration options. At this time, only booleans, enums, and explicit value lists are supported; for more complex configuration including arbitrary integers and floats, manual menu construction is needed.
```

Comment:
This should link the docs for v5 https://docs.bepinex.dev/api/BepInEx.Configuration.ConfigFile.html

Also is there a reason that numeric types with AcceptableValueRange are not supported as sliders